src/dune_rules/melange_stanzas.ml

```diff
@@ -47,6 +48,12 @@ module Emit = struct
     t
 
   let decode =
+    let extension_field name =
+      let+ loc, extension = located (field name ~default:"js" string) in
+      if String.starts_with ~prefix:"." extension then
```

Comment:
Should be `is_prefix` for compat.

Reply:
https://github.com/ocaml/dune/pull/6457/commits/5f2f51f500a08b17fb1e1a250dbb2cc742f110d3